docs/data/menu/main.yaml

```diff
@@ -0,0 +1,34 @@
+---
+main:
+  - name: Getting Started
+    ref: "/gettingstarted"
+    sub:
+      - name: Github
+        ref: /getting_started/github
+      - name: Gitlab
+        ref: /getting_started/gitlab
+      - name: Bitbucket
+        ref: /getting_started/bitbucket
+  - name: Providers
+    ref: "/providers"
+  - name: Configuration Providers
+    ref: "/config_providers"
+  - name: Examples
+    sub:
+    - name: IAM User to ingest AWS Logs into Splunk
```

Comment:
I think there's a missing tab for these 4 sub pages. 😄

Reply:
Are you referring to the "ref"? If so, that's omitted by design, the "Examples" heading is just a heading and contains no page, the child example pages are self explanatory.

Reply:
I was referring to the Example heading, which I misread as another page. My mistake!